<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuchungsDetails</class>
 <widget class="QWidget" name="BuchungsDetails">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buchungsdetails</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>13</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>55</y>
     <width>26</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Preis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>98</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>von</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>141</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>bis</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>13</y>
     <width>260</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>55</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>98</y>
     <width>84</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_2">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>141</y>
     <width>84</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>421</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab_flug">
    <attribute name="icon">
     <iconset resource="Graphic_Elements.qrc">
      <normaloff>:/icons/airplane.png</normaloff>:/icons/airplane.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Flug</string>
    </attribute>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>6</y>
       <width>20</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>von</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>41</y>
       <width>26</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nach</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>76</y>
       <width>34</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Airline</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>111</y>
       <width>69</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bookingclass</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fDest_Flug">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>6</y>
       <width>40</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tDest_Flug">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>41</y>
       <width>40</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_airline">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>76</y>
       <width>140</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_bookinglass">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>111</y>
       <width>140</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Flug_Von">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>6</y>
       <width>290</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Flug_Nach">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>41</y>
       <width>290</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_mietwagen">
    <attribute name="icon">
     <iconset resource="Graphic_Elements.qrc">
      <normaloff>:/icons/rentalcar.png</normaloff>:/icons/rentalcar.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Mietwagen</string>
    </attribute>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>4</y>
       <width>65</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Abhostation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>37</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ruecgabestation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Company</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>103</y>
       <width>63</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vehicleclass</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fromStation">
     <property name="geometry">
      <rect>
       <x>96</x>
       <y>4</y>
       <width>240</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_toStation">
     <property name="geometry">
      <rect>
       <x>96</x>
       <y>37</y>
       <width>240</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_company">
     <property name="geometry">
      <rect>
       <x>96</x>
       <y>70</y>
       <width>150</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_vehicleclass">
     <property name="geometry">
      <rect>
       <x>96</x>
       <y>103</y>
       <width>150</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_hotel">
    <attribute name="icon">
     <iconset resource="Graphic_Elements.qrc">
      <normaloff>:/icons/hotel.png</normaloff>:/icons/hotel.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Hotel</string>
    </attribute>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>4</y>
       <width>29</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hotel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>37</y>
       <width>28</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Stadt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>70</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Zimmertyp</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_hotel">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>4</y>
       <width>250</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_town">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>37</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_roomtype">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>70</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_zug">
    <attribute name="icon">
     <iconset resource="Graphic_Elements.qrc">
      <normaloff>:/icons/train.png</normaloff>:/icons/train.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Zug</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>1</y>
       <width>20</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>von</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>31</y>
       <width>40</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Abfahrt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>62</y>
       <width>26</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nach</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>92</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ankunft</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>123</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tickettyp</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>145</y>
       <width>25</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>über</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fDest_Zug">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>1</y>
       <width>190</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tDest_Zug">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>62</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tickettyp">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>123</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_abfahrt">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>31</y>
       <width>56</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_ankunft">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>92</y>
       <width>56</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_ueber">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>153</y>
       <width>221</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>166</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Graphic_Elements.qrc"/>
 </resources>
 <connections/>
</ui>
